tests/e2e/test_sampling_params.py

```diff
@@ -0,0 +1,237 @@
+import pytest
+from spyre_util import get_spyre_backend_list
+from vllm import LLM, SamplingParams
+
+pytestmark = pytest.mark.basic
+
+
+@pytest.fixture(scope="module")
+def spyre_model() -> LLM:
+    return LLM(model="ibm-granite/granite-3.1-2b-instruct", max_model_len=128)
+
+
+@pytest.fixture(scope="function", autouse=True)
+@pytest.mark.parametrize("backend", get_spyre_backend_list())
+def setenv(monkeypatch: pytest.MonkeyPatch):
+    monkeypatch.setenv("VLLM_SPYRE_DYNAMO_BACKEND", "eager")
+
+
+def test_spyre_batch1_temperature(spyre_model: LLM):
```

Comment:
also worth checking out `use_llm_cache` - https://github.com/vllm-project/vllm-spyre/blob/main/tests/e2e/test_spyre_cb.py#L25

We definitely need some sort of caching here